build.gradle

```diff
@@ -1,6 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.2.71'
```

Comment:
We should use only one version of kotlin across the app.

Reply:
right, removed